apps/drec-ui/src/utils/formatters/PowerFormatter.ts

```diff
@@ -2,7 +2,7 @@ import { Unit } from '@energyweb/utils-general';
 
 export class PowerFormatter {
     static readonly displayUnit: string = 'kWh';
-    static readonly capacityDisplayUnit: string = 'kW';
+    static readonly capacityDisplayUnit: string = 'W';
 
     static readonly decimalPlaces: number = 3;
```

Comment:
Changing display unit without `decimalPlaces` can lead to unpredictable results/bugs

Reply:
Agree but in this case I don't think it should afffect it as it doesn't convert from kw to w and vice-versa. It just handles digits. And capacity will always be in Watts.

Reply:
The reason it won't affect is that I have a different method "capacityFormatDisplay" which won't divide by the capacity unit.

Reply:
```static capacityFormatDisplay(powerInWatt: number, includeDisplayUnit?: boolean): string {
        return `${PowerFormatter.formatter.format(powerInWatt)}${```
   As far as I can see it is dividing by `PowerFormatter.formatter.format(`

Reply:
Yes. But in D-REC I use this for displaying Capacity which is in Watts but also Reads with care in KW.
What do you propose I should change here?

Reply:
The thing is if you use `capacityFormatDisplay` you will be formatting the amount dividing it by 1000, which should be used for KW, but you will be displaying W. 
This can lead to confusions, because on the UI you will see for e.g.
` 100 W `
while under the hood it will look like this 
`100000` watts received from backend - formatted to `100` W displayed on ui
So I guess if you want to display in Watts you should avoid any use of PowerFormatter in this case and just show (W) unit near the non-formatted amount received from backend.

Reply:
Oh, I see, I am still formatting the data. Sry, missed that.
Well, I can change the method to something like:
static capacityFormatDisplay(powerInWatt: number, includeDisplayUnit?: boolean): string {
        return `${powerInWatt}${
            includeDisplayUnit ? ' ' + PowerFormatter.capacityDisplayUnit : ''
        }`;
    }
This way it won't affect the data from the API and I don't need to change everywhere were I call this method.
What do you think?